src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -674,7 +689,7 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId }
                 <RadioGroup
                   row
                   value={formik.values.functionId}
-                  onChange={e => setOperationType(parseInt(e.target.value))}
+                  onChange={e => setOperationType(parseInt(e.target.value), setFId(e.target.value))}
```

Comment:
Why do we have a set having a set in its parameters?

Reply:
here  functionId change if sale or purchase